Dockerfile.backwards

```diff
@@ -19,7 +19,7 @@ ENV SETUPTOOLS_USE_DISTUTILS=stdlib
 RUN pip3 install --upgrade pip setuptools
 RUN pip3 install -r requirements.txt
 RUN pip3 install colcon-bundle
-
+RUN pip3 install colcon-common-extensions
```

Comment:
Why do you need to install this to support setup.bash?

Reply:
my understanding was colcon-common-extensions installs colcon-bash, which is needed to include bash files in the bundle.

Reply:
if we specifically need colcon-bash, I'd suggest just installing that rather than pulling in a grab bag of other packages